etc/kubernetes/helm/values.yaml

```diff
@@ -23,4 +23,16 @@ nfs:
   enabled: false
   internal_server_ip_address:
 
+ingress:
+  enabled: false
+  namespace: enterprise-gateway
```

Comment:
Since we're using `{{ .Release.Namespace }}` for the namespace, this entry can be removed.